File: JavaRewrite/src/main/java/io/github/deadshot465/shared/Question.java
package io.github.deadshot465.shared;

public interface Question {
    void question1();
    void question2();
    void question3();
    void question4();

    default void execute(int num) {
        switch (num) {
            case 1 -> question1();
            case 2 -> question2();
            case 3 -> question3();
            case 4 -> question4();
            default -> {
            }
        }
    }
}

File: KotlinRewrite/src/main/kotlin/io/github/deadshot465/shared/Question.kt
package io.github.deadshot465.shared

interface Question {
    fun question1()
    fun question2()
    fun question3()
    fun question4()

    fun execute(num: Int) {
        when (num) {
            1 -> question1()
            2 -> question2()
            3 -> question3()
            4 -> question4()
            else -> {}
        }
    }
}